### DISCUSSION TITLE
Background color of the option inside Select component - how to change

### DISCUSSION THREAD

**Discussion by belkocik** [chakra-ui/chakra-ui#5862] [Q&A] [answered]
Hello, how to change the background color of the select component inside option element when I hover it? A blue background on the image below.
![image](https://user-images.githubusercontent.com/62507500/163039921-13878f99-2458-4b61-ba7f-6281b8d5cc62.png)

**Comment by JTBrinkmann** [comment, marked as answer]
As already explained in the Discord chat
> you cannot freely style the browser-native select list. A few CSS styles apply, but most do not. It's also different from browser to browser. A solution would be to reimplement `<select>` using custom logic instead of relying on the browser's native UI (there are libraries for this). **Maybe Chakra's [`<Menu>`](https://chakra-ui.com/docs/components/overlay/menu) component also serves your needs.**

(note: instead of custom logic you could also do a CSS-only solution using radio inputs, there's non-Chakra-specific guides for this, just search for "styling html select option" or similar)

In case you want a more authoritive answer, here is the quote from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option#styling_with_css):
> Styling the <option> element is highly limited. Options don't inherit the font set on the parent. In Firefox, only [color](https://developer.mozilla.org/en-US/docs/Web/CSS/color) and [background-color](https://developer.mozilla.org/en-US/docs/Web/CSS/background-color) can be set however in Chrome or Safari it's not possible to set any properties. You can find more details about styling in [our guide to advanced form styling](https://developer.mozilla.org/en-US/docs/Learn/Forms/Advanced_form_styling).
